core/controllers/reader.py

```diff
@@ -387,7 +482,12 @@ class ExplorationCompleteEventHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """
+        Handles POST requests.
+        Args:
+            exploration_id: The str ID of the exploration the learner is
+                            finishing.
```

Comment:
Indent by 4 from the previous line; keep things consistent within this file and with other files.